### PR TITLE
Change the survey to a call for a user interview

### PR DESCRIPTION
**Pull request by Tbaut**
We figured it'd be good to have more interviews

### CODE REVIEW

File: packages/files-ui/src/locales/de/messages.po [hunk 1]
```diff
@@ -550,6 +547,9 @@ msgstr "Gespeicherte Browser"
 msgid "Saved on:"
 msgstr "Gesichert am:"
 
+msgid "Schedule a 15min call"
```
Comment (asnaith) [inline packages/files-ui/src/locales/de/messages.po]:
```suggestion
msgid "Schedule a 15 min call"
```

File: packages/files-ui/src/locales/en/messages.po [hunk 2]
```diff
@@ -553,6 +550,9 @@ msgstr "Saved Browsers"
 msgid "Saved on:"
 msgstr "Saved on:"
 
+msgid "Schedule a 15min call"
```
Comment (asnaith) [inline packages/files-ui/src/locales/en/messages.po]:
```suggestion
msgid "Schedule a 15 min call"
```

File: packages/files-ui/src/locales/en/messages.po [hunk 3]
```diff
@@ -553,6 +550,9 @@ msgstr "Saved Browsers"
...
+msgstr "Schedule a 15min call"
```
Comment (asnaith) [inline packages/files-ui/src/locales/en/messages.po]:
```suggestion
msgstr "Schedule a 15 min call"
```

File: packages/files-ui/src/locales/es/messages.po [hunk 4]
```diff
@@ -554,6 +551,9 @@ msgstr "Navegadores guardados"
 msgid "Saved on:"
 msgstr "Guardado el:"
 
+msgid "Schedule a 15min call"
```
Comment (asnaith) [inline packages/files-ui/src/locales/es/messages.po]:
```suggestion
msgid "Schedule a 15 min call"
```

File: packages/files-ui/src/locales/fr/messages.po [hunk 5]
```diff
@@ -554,6 +551,9 @@ msgstr "Navigateurs enregistrés"
 msgid "Saved on:"
 msgstr "Enregistré sur :"
 
+msgid "Schedule a 15min call"
```
Comment (asnaith) [inline packages/files-ui/src/locales/fr/messages.po]:
```suggestion
msgid "Schedule a 15 min call"
```

File: packages/files-ui/src/locales/no/messages.po [hunk 6]
```diff
@@ -550,6 +547,9 @@ msgstr ""
 msgid "Saved on:"
 msgstr ""
 
+msgid "Schedule a 15min call"
```
Comment (asnaith) [inline packages/files-ui/src/locales/no/messages.po]:
```suggestion
msgid "Schedule a 15 min call"
```

File: packages/files-ui/src/Components/SurveyBanner.tsx [hunk 7]
```diff
@@ -77,13 +77,13 @@ const SurveyBanner = ({ onHide }: Props) => {
...
         <Trans>
-          Help us improve File in less than 1 minute.
+          Want to help shape this product?
         </Trans>
         <span
           className={classes.link}
           onClick={onOpen}
         >
-          <Trans>Continue</Trans>
+          <Trans>Schedule a 15min call</Trans>
```
Comment (asnaith) [inline packages/files-ui/src/Components/SurveyBanner.tsx]:
```suggestion
          <Trans>Schedule a 15 min call</Trans>
```

**Review by asnaith — APPROVED** [review]
Looks good, works as intended. I left some suggestions for a space character after the "15" but up to you :)

**Review by tanmoyAtb — APPROVED** [review]
👍

**Pull request merged**